openedx/adg/admin_dashboard/admin_settings/views.py

```diff
@@ -276,3 +276,39 @@ def get_queryset(self):
             )
 
         return invitations
+
+
+class UpdatePermissions(ListView):
+    """
+    UpdatePermissions View
+    """
+    def get(self, request, *args, **kwargs):
+        """
+        It will get invitation object of the user and give permissions which
+        are assigned to that user in invitation object
+
+        Returns:
+            It will redirect to admin dashboard if user is active and have permissions
+            otherwise it will redirect to home page
+        """
+        user = self.request.user
```

Comment:
how about before granting any kind of permission, Maybe we can remove all the groups if there are any then this view can be used for update as well?